### PR TITLE
Add pagination to `/db/*`

### PR DESCRIPTION
**Pull request by AI-Mozi**
#21

### CODE REVIEW

**Comment by AI-Mozi** [comment]
I would personally make footer's top margin smaller or even 0

File: views/db/advisories/index.erb [hunk 1]
```diff
@@ -12,3 +12,5 @@
 <% @advisories.each do |advisory| %>
   <p><a href="/db/advisories/<%=h advisory.id %>"><%=h advisory %></a></p>
 <% end %>
+
+<%= pagy_bulma_nav(@pagy) if @pagy.pages > 1 %>
```
Comment (postmodern) [inline views/db/advisories/index.erb:16]: This tiny bit of logic should be extracted into a partial, that way if we want to add to or change the pagination, we can change it in one place.

File: app.rb [hunk 2]
```diff
@@ -770,4 +778,13 @@ class App < Sinatra::Base
     erb :about
   end
 
+  private
+
+  def pagy_get_vars(collection, vars)
```
Comment (postmodern) [inline app.rb:800]: Needs minimal documentation that explains this is used for the pagination defaults.

File: app.rb [hunk 3]
```diff
@@ -65,12 +65,16 @@
 
 require 'ronin/app/version'
 
+# others
+require 'pagy'
+require 'pagy/extras/bulma'
```
Comment (postmodern) [inline app.rb:71]: Add an extra newline after the `require`.

**Review by postmodern — CHANGES_REQUESTED** [review]
Minor documentation changes needed, and probably should extract the pagination view logic out into it's own partial template.

**Pull request merged**